test/commands/autocomplete/create.test.ts

```diff
@@ -39,6 +39,14 @@ skipWindows('Create', () => {
       expect(cmd.bashSetupScript).to.eq(`OCLIF_EXAMPLE_AC_BASH_COMPFUNC_PATH=${config.cacheDir}/autocomplete/functions/bash/oclif-example.bash && test -f $OCLIF_EXAMPLE_AC_BASH_COMPFUNC_PATH && source $OCLIF_EXAMPLE_AC_BASH_COMPFUNC_PATH;\n`)
     })
 
+    it('#bashSetupScript wiht more tahn one dash', async () => {
+      const rootWithDash = path.resolve(__dirname, '../../assets/package.json')
+      const confitWithDash = new Config({root:rootWithDash})
+      await confitWithDash.load()
```

Comment:
remove `assets/package.json` and instead create a new empty `Config` object, then set `bin` to `oclif-cli-example` after loading it, see:
https://github.com/oclif/plugin-autocomplete/blob/main/test/commands/autocomplete/create.test.ts#L92-L95

Reply:
Fixed!